.travis/wheel.test.sh

```diff
@@ -6,7 +6,7 @@ run_test() {
   entry=$1
   CPYTHON_VERSION=$($entry -c 'import sys; print(str(sys.version_info[0])+str(sys.version_info[1]))')
   (cd wheelhouse && $entry -m pip install tensorflow_io-*-cp${CPYTHON_VERSION}-*.whl)
-  $entry -m pip install -q pytest boto3 google-cloud-pubsub==0.39.1 pyarrow==0.14.1 pandas==0.24.2 scikit-learn==0.20.4
+  $entry -m pip install -q pytest boto3 google-cloud-pubsub==0.39.1 pyarrow==0.14.1 pandas==0.24.2 scikit-learn==0.20.4 gast==0.2.2
```

Comment:
We need to pin gast to 0.2.2 here and elsewhere because the current version of tf we are using did not pin this dependency appropiately causing problems: https://github.com/tensorflow/tensorflow/issues/32448#issuecomment-530628039 . This should be fixed in the next tensorflow RC! Let me know if there's a better way for us to pin this. Have we thought about spinning up a requirements.txt to hold these dependencies and pins?

Reply:
@yongtang let me know if you prefer a better way of pinning this dep! Thanks!

Reply:
@suyashkumar Thanks! I think that should be fine. We can always update later.

The tensorflow-io only depends on tensorflow' python requirement with no extra dependency. If needed we could add the requirement.txt in `tests/` directory.

Reply:
Ah, I see! Whenever we update our tensorflow dependency to `v2.0.0-rc1` (released 4 days ago) we should be okay without this, because they've pinned `gast` in that release https://github.com/tensorflow/tensorflow/commit/16cad8c1f6f0180b0de20abf117ed64f42f07fb6 . I imagine we will probably do that soon, and so I agree that this is likely fine for now (though if anyone uses this in the interim, they will have to pin gast or tensorflow on their own I think).